improver/psychrometric_calculations/hail_size.py

```diff
@@ -458,6 +546,8 @@ def hail_size_data(
                 Cube of cloud condensation level pressure
             humidity_mixing_ratio_at_268
                 Cube of humidity mixing ratio at the pressure of the environment at 268.15K
+            wet_bulb_zero
+                Cube of the height of the wet-bulb freezing level (m)
```

Comment:
If you put the units here i.e. the "(m)" then it should probably be something like. "Convertible to metres.", as it doesn't actually have to be in metres, as the `_check_cubes` method could do a conversion.

Reply:
I've just removed the metres here as its not needed and more consistent with the other cubes.